src/css/_app.scss

```diff
@@ -63,6 +63,9 @@
 
     .card {
         border: 1px solid map-get($theme-colors, "card-border");
+        & > div:first-child {
```

Comment:
Can you make this more specific to the layer controller? `.card` is used by every component since it is in TitleInfo